### ISSUE TITLE
Missing link name: "Close Menu" menu-backer navigation control

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#19]
As with Freedom ([omeka-s-themes/freedom#39](https://github.com/omeka-s-themes/freedom/issues/39)), the "Close Menu" link on the menu drawer is missing a name.

**Comment by nelsonamaya82** [comment]
@aviannamiller this is fixed in the `master` branch, please double check and let me know if it works as expected.

**Comment by aviannamiller** [comment]
Thanks for the work on this. I am now having the same issue as https://github.com/omeka-s-themes/freedom/issues/39#issuecomment-2998206019 (fully outlined there, but to sum it up a bit: the nav control and toggle now do have accessible names, yet the toggle is not meshing well with its use of `aria-labelledby` to refer to `menu-backer`, thus giving an improper screen reader announcement on a collapsed toggle)

**Comment by nelsonamaya82** [comment]
@aviannamiller I've just applied all the adjustments we applied and tested on Lively Classic, so it now works exactly the same as in Classic. Please retest and let me know if it works.
